test/WebJobs.Script.Tests.Integration/Host/SecretsRepositoryTests.cs

```diff
@@ -370,6 +369,133 @@ public async Task GetSecretSnapshots_ReturnsExpected(SecretsRepositoryType repos
             }
         }
 
+        [Theory]
+        [InlineData(SecretsRepositoryType.BlobStorage)]
+        [InlineData(SecretsRepositoryType.BlobStorageSas)]
+        public async Task BlobRepository_WriteAsync_DoesNot_ClearBlobContents(SecretsRepositoryType repositoryType)
```

Comment:
With the previous code in `WriteToBlobAsync()`, this test failed 100% of the time locally because it would find null secrets when Reading.